plugins/variants/src/VcfTabixAdapter/VcfTabixAdapter.ts

```diff
@@ -56,6 +56,19 @@ export default class extends BaseFeatureDataAdapter {
     return this.vcf.getReferenceSequenceNames(opts)
   }
 
+  async getInfo() {
+    const header = await this.vcf.getHeader()
+    return header
+      .split('\n')
+      .filter(line => line.startsWith('##'))
+      .map(line => {
+        const str = line.slice(2)
+        const index = str.indexOf('=')
+        const [tag, data] = [str.slice(0, index), str.slice(index + 1)]
+        return { tag, data }
+      })
```

Comment:
For VCF, this should probably use `parser.getMetaData()`. That will include additional data inferred by the VCF parser that is not in the header (e.g. VCF has some built-in INFO field types that don't have to be specified in the header, but will be returned by `parser.getMetaData()`).

Reply:
I feel like I should include "just what's in the header". I can see the results of this getMetadata with the sort of default INFO field types and it is interesting but I dunno if it needs reporting

Reply:
There are times, though, that I think we'll want to get all the metadata. So maybe we need two different RPC calls? In that case `getInfo()` is perhaps a bit too generic of a name. Maybe `getHeaderData()` and `getMetadata()`?

Reply:
I might just call it getHeader and getMetadata and then ya

Note that regarding  "getting info for feature detail widgets" e.g. maybe tooltip for what a given detail field is on mouseover, we might need to figure out what that means. This is an asynchronous fetch for the metadata, that might want to be done prior to showing the feature detail panel, so then we can refresh page, metadata is shown, etc.